lib/src/intl_suggestors/intl_messages.dart

```diff
@@ -121,9 +121,13 @@ Attempting to add a different message with the same name:
     }
   }
 
+  /// The dependency analyzer thinks we depend on this if we have the literal string, so
```

Comment:
Jumping through some hoops here to avoid a spurious dependency on a Workiva internal package.